analyzer/restapi-plugin/src/main/java/eu/fasten/analyzer/restapiplugin/mvn/api/PackageApi.java

```diff
@@ -93,5 +93,7 @@ public PackageApi(PackageApiService service) {
                                        @RequestParam(required = false) Long releaseDate) {
         return service.getERCGLink(packageName, version, artifactRepo, releaseDate);
     }
+
```

Comment:
Could you please remove these two blank lines?

Reply:
Done